spec/factories/iiif_resource.rb

```diff
@@ -0,0 +1,23 @@
+FactoryBot.define do
+  factory :iiif_resource, class: IIIFResource do
+    transient do
+      # Fixture to use for stubbing the manifest.
+      manifest_fixture { nil }
+      # Source metadata identifier to stub out
+      source_metadata_identifier { nil }
+      # Reference to the spec to enable the factory to stub out requests.
+      spec { nil }
```

Comment:
define the stub in the factory, never thought of this; seems handy.

Reply:
Yeah, I'm not sure I'm a fan yet, but if we're going to try using more factories it seems like something to fiddle with.